lib/appbundler/app.rb

```diff
@@ -116,14 +116,15 @@ def binstub(bin_file)
     end
 
     def load_statement_for(bin_file)
+      name, version = app_spec.name, app_spec.version
+      bin_basename = File.basename(bin_file)
       <<-E
-bin_dir = File.dirname(__FILE__)
-if File.symlink?(__FILE__)
-  bin_dir = File.dirname(File.readlink(__FILE__))
-end
+gem "#{name}", "= #{version}"
```

Comment:
Is this one needed? Shouldn't it be written out in the massive bunch of `gem` commands that pin the world?